rust-mode.el

```diff
@@ -1317,9 +1318,11 @@ This is written mainly to be used as `end-of-defun-function' for Rust."
   (setq-local parse-sexp-lookup-properties t)
   (setq-local electric-pair-inhibit-predicate 'rust-electric-pair-inhibit-predicate-wrap)
   (add-hook 'after-revert-hook 'rust--after-revert-hook 'LOCAL)
```

Comment:
Yup - that was a mistake.  Thanks for catching it.  Fortunately the hook would usually be fairly harmless when run in other modes...but much better if it's local as intended.